src/utils/application.data.ts

```diff
@@ -118,6 +118,18 @@ export const getFromApplicationData = (req: Request, key: string, id: string, er
   }
 };
 
+export const hasAnyBosWithTrusteeNoc = (session: Session | undefined) => {
+  const appData = getApplicationData(session);
+  const boWithTrusteeNocFilter = bo => bo.trustees_nature_of_control_types?.length > 0 || false;
```

Comment:
You may not need the "|| false" as undefined > 0 is false

Reply:
Ah fair point - I thought the `?.` would terminate evaluation on the whole comparison: `bo.trustees_nature_of_control_types?.length > 0` and not just referencing the length. Will take that out.